site/cypress/e2e/Cypress-Gradeable/late_submission_warning_messages.spec.js

```diff
@@ -118,43 +129,82 @@ const SubmitAndCheckMessage = (gradeable_type, upload_file1, invalid_late_day, v
     cy.logout();
 };
 
-it('Checks whether daylight savings warning message should be appearing at current date.', () => {
-    /*
-    First we set the date to a date close to the spring daylight savings time and then check that the message appears.
-    Then we change the date to a month later, and check that the "daylight" message doesn't appear.
-    */
-    const currentDate = new Date();
-    const futureDate = new Date(currentDate);
-    futureDate.setDate(currentDate.getDate() + 7);
+const checkDaylightBanner = (late_days, existence) => {
+    cy.login('instructor');
+    cy.visit(['sample', 'gradeable', daylight_gradeable, 'update?nav_tab=5']);
+    cy.get('#late_days')
+        .clear()
+        .type(late_days)
+        .type('{enter}');
```

Comment:
This along with a lot of the chained .type and .clear commands will be linted in a one of Dorian's PRs, and it will fail. You should make this not chained to not add more work to that PR.